pkg/storage/test/storage.go

```diff
@@ -31,11 +31,16 @@ func RunAllTests(t *testing.T, ds storage.OpenFGADatastore) {
 	})
 	// Tuples.
 	t.Run("TestTupleWriteAndRead", func(t *testing.T) { TupleWritingAndReadingTest(t, ds) })
-	t.Run("TestTuplePaginationOptions", func(t *testing.T) { TuplePaginationOptionsTest(t, ds) })
 	t.Run("TestReadChanges", func(t *testing.T) { ReadChangesTest(t, ds) })
 	t.Run("TestReadStartingWithUser", func(t *testing.T) { ReadStartingWithUserTest(t, ds) })
-	t.Run("TestRead", func(t *testing.T) { ReadTest(t, ds) })
-	t.Run("TestReadPage", func(t *testing.T) { ReadPageTest(t, ds) })
+
+	// TODO I suspect there is overlap in test scenarios. Consolidate them into one
+	t.Run("ReadPageTestCorrectnessOfContinuationTokens", func(t *testing.T) { ReadPageTestCorrectnessOfContinuationTokens(t, ds) })
```

Comment:
I see the comment, are we going to collapse these into simply `t.Run("ReadPage_TestCorrectnessOfContinuationTokens", ...)` with subtests?

Reply:
Yeah I think we should